package.json

```diff
@@ -12,7 +12,7 @@
     "watch:start": "nodemon src/server.ts",
     "migrate": "ts-node ./migrations/migrate.ts",
     "migrate:docker": "./migrations/node-pg-migrate --database-url-var CONNECTION_STRING --migration-file-language ts --migrations-dir /app/migrations --ignore-pattern '\\..*|.*migrate(.ts)?'",
-    "seed": "cp -r ./scripts/seed/*_data ./dist/scripts/seed && node ./dist/scripts/seed/index.js",
+    "seed": "npm run build && cp -r ./scripts/seed/*_data ./dist/scripts/seed && node ./dist/scripts/seed/index.js",
```

Comment:
I'm not sure about building the project here as we run the seed every time [we deploy a new version](https://github.com/decentraland/builder-server/blob/master/entrypoint.sh#L4). At that point, the project was already built.

Moreover, we should find a way to prevent migration and running the seed every time the project is deployed.

Reply:
Hm you're right. Thought about it when adding the build here, but the build was fast so didn't make much of it. It's weird to have the local project, run seed and it exploding, but I agree that this is not the solution